lib/bip_claims/service.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module BipClaims
+  class Service < Common::Client::Base
+    STATSD_KEY_PREFIX = 'api.bip_claims'
+    include Common::Client::Monitoring
+
+    configuration BipClaims::Configuration
+
+    def veteran_attributes(claim)
+      case claim.form_id
+      when '21P-530'
+        ssn, full_name, bday = claim.parsed_form.values_at(
+          'veteranSocialSecurityNumber',
+          'veteranFullName',
+          'veteranDateOfBirth'
+        )
+      end
+
+      OpenStruct.new(
+        ssn: ssn,
+        first_name: full_name['first'],
+        last_name: full_name['last'],
+        birth_date: bday
+      )
+    end
+
+    def lookup_veteran_from_mvi(claim)
+      MVI::AttrService.new.find_profile(veteran_attributes(claim))&.profile
+    rescue MVI::Errors::Base
+      nil
+    end
+
+    def create_claim(form_data)
+      veteran_record = lookup_veteran_from_mvi(form_data)
+      claimant_record = false # TODO: look up from MVI
+      return false unless veteran_record && claimant_record
```

Comment:
```suggestion
      return true if veteran_record && claimant_record
```
Just a thought to clarify double-negative "return false if not both" to "return true if both"

Reply:
makes sense; though in this context I'm trying to have the method return early with a false value, so that the submit_claim API call never occurs

Reply:
Yeah, if I'm reading it right, he wants to
```ruby
submit_claim(veteran: veteran_record, claimant: claimant_record) if veteran_record && claimant_record
```
So for clarity the last few lines of the method could be
```ruby
return submit_claim(veteran: veteran_record, claimant: claimant_record) if veteran_record && claimant_record
false
```
or 
```ruby
if veteran_record && claimant_record
  submit_claim(veteran: veteran_record, claimant: claimant_record) 
else
  false
end
```
What's most clear?

Reply:
ah ok, ignore my suggestion please